api/services/call_log_service.py

```diff
@@ -30,6 +27,11 @@ def handle_flow_run_details(self, jsonData):
             jsonData (json): json data that we are getting from webhook
         """
         try:
+            if "run_uuid" in jsonData:
+                self.flow_run_uuid = helpers.fetch_by_key(
+                    "run_uuid", jsonData
+                )  # Keeping this code to establish backward campatibility
+
```

Comment:
Can we reduce some processing here?
We need `run_uuid` only when we don't have `flow_run_details` in any older RapidPro flow webhook payload. So instead of running it everytime, let's run the code block only if `flow_run_details` is not available. Thoughts?

```py
if "flow_run_details" in jsonData:
  # do things here
elif "run_uuid" in jsonData:
  # do things here
  # For backward compatibility
```